site/signin.html.jet

```diff
@@ -8,7 +8,10 @@
   <main id="main" class="page form-page">
     <div class="page-header">
       <h1>{{ i18n("signin_page_header") }}</h1>
-      <p><a href="{{ routeToPath("/forgotpassword.html") }}">{{ i18n("signin_page_forgotpassword")}}</a></p>
+      {{forgot_password_link := config("forgot_password_link")}}
+      {{if forgot_password_link != "hide"}}
```

Comment:
Not 100% sure on this one. It's a bit of magic that people have to know. I hesitate to say make another config as there are already heaps, but at least then it would be more specific.

Reply:
just to clarify, have a feature toggle to hide forgot password, i.e. Users > Feature toggle: disable_forgot_password, and then have the User > Config: forgot_password_link if they want to override the default url?

Reply:
I personally think that is cleaner. @sam-shift72 thoughts?

Reply:
I don't have strong feelings either way. I can see the argument for a single config, but I don't think it'd be significantly harder to introduce a toggle for hiding it, e.g. `hide_forgot_password` and `forgot_password_url`.

Reply:
Looks like https://github.com/shift72/core-template/pull/494 has the same rule (`hide` or the url). If Sam is happy that's good enough for me. Will just need to ensure the docs are really clear this is an option.

Reply:
The pattern of having a feature toggle that disables something when on seems wrong.

Reply:
interestingly there is already a user config documented "password_reset_disabled" although its not used in core template or relish. its only used for cilents with custom template. I'm happy to have two config or toggles to disable / edit the link. I just need to know which way to do it so we can close of these PRs

Reply:
It makes sense when it's an exception to the default behaviour, as the default value will be false. Our feature toggle/config system leaves a lot to be desired...